src/App.js

```diff
@@ -51,7 +49,7 @@ const theme = createTheme({
 
 export default function App() {
   const [devMode, setDevMode] = React.useState(getIsDevMode);
-  const visiblePages = getVisiblePages();
+  const [visiblePages, setVisiblePages] = React.useState(getVisiblePages);
```

Comment:
You need a setter, to be able to update the value. othersiwe you will not ba able to modify this value